api/routes/apds/get.js

```diff
@@ -0,0 +1,41 @@
+const logger = require('../../logger')('users route get');
+const defaultApdModel = require('../../db').models.apd;
+const defaultUserModel = require('../../db').models.user;
+const loggedIn = require('../../auth/middleware').loggedIn;
+
+module.exports = (
+  app,
+  ApdModel = defaultApdModel,
+  UserModel = defaultUserModel
+) => {
+  logger.silly('setting up GET /apds route');
+
+  app.get('/apds', loggedIn, async (req, res) => {
+    logger.silly(req, 'handling GET /apds');
+
+    try {
+      // TODO: we should have a helper method that fetches
+      // and returns state/stateId given userId
```

Comment:
I suppose there's no reason we couldn't attach the user model directly to the request object.  Then if we add a helper method to the user model class, we could have something like `await req.user.model.getState()`?

Reply:
yes, i like that!